mysql-test/mysql-test-run.pl

```diff
@@ -194,7 +194,7 @@ sub set_term_args {
 
 # If you add a new suite, please check TEST_DIRS in Makefile.am.
 #
-my $DEFAULT_SUITES= "main,sys_vars,binlog,binlog_encryption,rpl_encryption,encryption,"
+my $DEFAULT_SUITES= "main,sys_vars,binlog,binlog_encryption,binlog_gtid,rpl_encryption,encryption,"
```

Comment:
Not sure I like introducing new `binlog_gtid` suite in 5.7 - it's more like 8.0 refactoring feature.
I am not saying it does not make sense, but in order to stick closer to the upstream code I wouldn't do this

Reply:
I decided to introduce new suite, because this tests ideally fits binlog_gtid suite that is present in 8.0. Of course I can move it somewhere else. Please let me know if you want me to do this.

Reply:
Moved to suite/binlog as all binlog_gtid related tests are there as well in 5.7. For 8.0 the test will go to suite/binlog_gtid.